bindings/matrix-sdk-ffi/src/utils.rs

```diff
@@ -15,9 +15,20 @@
 use std::{mem::ManuallyDrop, ops::Deref};
 
 use async_compat::TOKIO1 as RUNTIME;
-use ruma::UInt;
+use ruma::{MilliSecondsSinceUnixEpoch, UInt};
 use tracing::warn;
 
+#[derive(Debug, Clone)]
+pub struct Date64(u64);
```

Comment:
I would rename `Date64` to `Timestamp` since it represents a timestamp everywhere this new type is used.

Reply:
Amended the PR with this new name, agree its much clearer.